Assets/Scripts/DataModels/Missions/MissionOutcomes/MoneyOutcome.cs

```diff
@@ -6,7 +6,7 @@ public class MoneyOutcome : MissionOutcome
     public float moneyMin;
     public float moneyMax;
 
-    public override void Process()
+    public override void Process(Mission mission)

```

Comment:
Yeah I wanted to overload this and have a parameterless one. But couldn't find a clean way of determining which method to call in Mission.ProcessOutcome

Reply:
yeah this is fine for now